pkg/lightning/backend/local/local.go

```diff
@@ -165,6 +168,7 @@ func (f *importClientFactoryImpl) makeConn(ctx context.Context, storeID uint64)
 		addr = store.GetAddress()
 	}
 	opts = append(opts,
+		grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(unlimitedRPCRecvMsgSize)),
```

Comment:
have you test it manually?

Reply:
I ran and passed the lightning_duplicate_detection integration test in local, which covered the modified code.

Reply:
I mean to reproduce a scenario in the issue that have many duplicates and TiKV send a grpc message larger that some size, say 32MiB

Reply:
No. But I have tried to reduce MaxCallRecvMsgSize e.g. 200KB which did trigger the expected error.

I will try to compose a new integration test that will send 4MiB+ KV back.

Reply:
o, that should be enough, please also update this in the PR description